ocs_ci/utility/svt.py

```diff
@@ -0,0 +1,51 @@
+import os
+import logging
+import shutil
+from ocs_ci.utility.utils import run_cmd
+
+log = logging.getLogger(__name__)
+
+
+def svt_project_clone():
+    run_cmd("git clone https://github.com/openshift/svt.git /tmp/svt")
+
+
+def svt_create_venv_setup():
+    run_cmd("virtualenv -p /bin/python2 /tmp/venv")
+    run_cmd("/bin/sh -c 'source /tmp/venv/bin/activate && python --version'")
+    run_cmd("/bin/sh -c 'source /tmp/venv/bin/activate && pip install -r registry_requirement.txt'")
+
+
+def svt_cluster_loader(clusterload_file="/tmp/svt/openshift_scalability/config/master-vert.yaml"):
+    KUBECONFIG = os.getenv('KUBECONFIG')
+    """
+    https://github.com/openshift/svt/blob/master/openshift_scalability/README.md
+
+    Arguments:
+        -f : This is the input config file used to define the test.
+        -kubeconfig : kebeconfig path
+    Returns:
+        None
+    """
+    cwd = os.getcwd()
+    os.chdir('/tmp/svt/openshift_scalability/')
+    cmd = f"/bin/sh -c 'source /tmp/venv/bin/activate && python /tmp/svt/openshift_scalability/cluster-loader.py \
+    -f {clusterload_file} --kubeconfig {cwd}/{KUBECONFIG}'"
+    run_cmd(cmd)
+    os.chdir(cwd)
+
+
+def svt_cleanup():
```

Comment:
@vikasmulaje, Did this remove the namespace and pods created by SVT ?

Reply:
Updated code with removing created projects